### ISSUE TITLE
Keys in headers are not replaced

### ISSUE THREAD

**Issue by sidkdbl07** [ivanbicalho/python-docx-replace#4]
I placed a key (e.g., ${Title} ) both in the body of the document an in the header. The instance in the body of the document was replaced, but the instance of the header was not.

**Comment by ivanbicalho** [comment]
You are right, there was a bug because the program wasn't getting the header/footer paragraphs. It's now solved in the version `0.4.1`, by including the following code: 

```python
for section in doc.sections:
    for paragraph in section.header.paragraphs:
        paragraphs.append(paragraph)
    for paragraph in section.footer.paragraphs:
        paragraphs.append(paragraph)
```

Can you check if it's ok? If so, we can close this, thanks!

**Comment by ivanbicalho** [comment]
Fixed in version `0.4.1`
